From 452c52b11a3a8e54b9e63397c9fcd645f3a3a6fb Mon Sep 17 00:00:00 2001
From: Jason Cawood <jasoncawood@gmail.com>
Date: Thu, 24 Oct 2019 16:22:55 -0700
Subject: [PATCH 09/25] Update u2f.conf.sample

This is to match the changes from commit bf3a625d846e22ff0ac3abac386781e0c09ffcab

Signed-off-by: Gustavo B. Schenkel <gustavo.schenkel@gmail.com>
---
 u2f.conf.sample | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/u2f.conf.sample b/u2f.conf.sample
index 7e685d0..040b5c2 100644
--- a/u2f.conf.sample
+++ b/u2f.conf.sample
@@ -6,7 +6,7 @@ notify 100 {
 	match "subsystem"	"DEVICE";
 	match "type"		"ATTACH";
 	match "vendor"		"0x1050";
-	match "product"		"(0x0113|0x0114|0x0115|0x0116|0x0120|0x0200|0x0402|0x0403|0x0406|0x0407|0x0410)";
+	match "product"		"(0x0113|0x0114|0x0115|0x0116|0x0120|0x0120|0x0200|0x0402|0x0403|0x0406|0x0407|0x0410)";
 	action	"chgrp u2f /dev/$cdev; chmod g+rw /dev/$cdev";
 };
 
-- 
2.32.0

